packages/features/tasker/tasks/scanWorkflowBody.ts

```diff
@@ -95,28 +108,14 @@ export async function scanWorkflowBody(payload: string) {
           verifiedAt: new Date(),
         },
       });
-      continue;
     }
-
-    const isSpam = await iffyScanBody(workflowStep.reminderBody, workflowStep.id);
-
-    if (isSpam) {
-      if (!workflowStep.workflow.user?.whitelistWorkflows) {
-        // We won't delete the workflow step incase it is flagged as a false positive
-        log.warn(`Workflow step ${workflowStep.id} is spam with body ${workflowStep.reminderBody}`);
-        await lockUser("userId", userId.toString(), LockReason.SPAM_WORKFLOW_BODY);
-
-        // Return early if spam is detected
-        return;
-      }
-      log.warn(
-        `For whitelisted user, workflow step ${workflowStep.id} is spam with body ${workflowStep.reminderBody}`
-      );
-    }
-
-    await prisma.workflowStep.update({
+  } else {
```

Comment:
is this else for `if (process.env.IFFY_API_KEY)`?

can we just do  `if (!process.env.IFFY_API_KEY)` instead to make it more readable?